backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ProjectSummary.java

```diff
@@ -53,6 +53,10 @@ protected static void setDefaultFields(Project document, Project copy) {
         copyField(document, copy, _Fields.DESCRIPTION);
         copyField(document, copy, _Fields.VERSION);
         copyField(document, copy, _Fields.CLEARING_TEAM);
+        copyField(document, copy, _Fields.PROJECT_RESPONSIBLE);
+        copyField(document, copy, _Fields.PROJECT_OWNER);
+        copyField(document, copy, _Fields.SECURITY_RESPONSIBLES);
+        copyField(document, copy, _Fields.ENABLE_VULNERABILITIES_DISPLAY);
```

Comment:
No changes are needed in this file (ProjectSummary). Kindly revert this change.

Reply:
done.